ambry-server/src/main/java/com/github/ambry/server/AmbryServer.java

```diff
@@ -312,17 +312,16 @@ public void startup() throws InstantiationException {
       }
 
       // Other code
-      List<AmbryHealthReport> ambryHealthReports = new ArrayList<>();
+      List<AmbryStatsReport> ambryStatsReports = new ArrayList<>();
       Set<String> validStatsTypes = new HashSet<>();
       for (StatsReportType type : StatsReportType.values()) {
         validStatsTypes.add(type.toString());
       }
       if (serverConfig.serverStatsPublishHealthReportEnabled) {
```

Comment:
nit: you may wants to rename the config file to serverStatsPublishStatsReportEnabled?